manifests/03-rbac-role-ns-openshift-config-managed.yaml

```diff
@@ -0,0 +1,21 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: console-operator
+  namespace: openshift-config-managed
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - services
+  - events
+  - configmaps
+  - secrets
+  - pods
```

Comment:
Def should not need `pods`, guessing not `events`.  And we are just `watch`ing, so full CRUD is too much.

Reply:
done